test/unittests/iotools/converters/aibl_to_bids/test_aibl_utils.py

```diff
@@ -1,6 +1,73 @@
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
 import pytest
 
 
+@pytest.mark.parametrize(
+    "visit, visit_list, date_list, expected",
+    [
+        ("bl", ["bl", "m10"], ["01/01/2000", -4], None),
+        ("m10", ["bl", "m10"], ["01/01/2000", -4], "11/01/2000"),
+        ("m0006", ["bl"], ["01/01/2000"], "07/01/2000"),
+    ],
+)
+def test_compute_exam_date_from_baseline_success(
+    visit, date_list, visit_list, expected
+):
+    from clinica.iotools.converters.aibl_to_bids.utils.clinical import (
+        _compute_exam_date_from_baseline,
+    )
+
+    assert _compute_exam_date_from_baseline(visit, date_list, visit_list) == expected
+
+
+def test_compute_exam_date_from_baseline_raiseValue():
+    from clinica.iotools.converters.aibl_to_bids.utils.clinical import (
+        _compute_exam_date_from_baseline,
+    )
+
+    with pytest.raises(
+        ValueError,
+        match=f"Unexpected visit code foo. Should be in format mX :"
+        "Ex: m0, m6, m12, m048...",
+    ):
+        _compute_exam_date_from_baseline("foo", [], [])
+
+
+def test_find_exam_date_in_other_csv_files():
```

Comment:
Are you planing to implement these in this PR ?

If not, please remove them otherwise we would see those tests passing while the related code isn't actually tested.